code/modules/events/spider_infestation.dm

```diff
@@ -25,13 +25,13 @@
 
 /proc/create_midwife_eggs(amount)
 	var/list/spawn_locs = list()
-	for(var/x in GLOB.xeno_spawn)
+	for(var/x in GLOB.generic_maintenance_landmark)
 		var/turf/spawn_turf = x
 		var/light_amount = spawn_turf.get_lumcount()
 		if(light_amount < SHADOW_SPECIES_LIGHT_THRESHOLD && is_safe_turf(spawn_turf))
```

Comment:
i know this isn't the place to balance it but lmao did they literally just copypaste the nightmare version of this check a bit too hard? no reasons for spiders to always be in a dark place (or should at least rename the macro to not be so 🤨)

(don't change this here, jus twanted to bring it up since i noticed it)